src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -194,7 +170,6 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
         onSubmitButtonPress(currentAmount);
     }, [onSubmitButtonPress, currentAmount]);
 
-    const formattedAmount = MoneyRequestUtils.replaceAllDigits(currentAmount, toLocaleDigit);
```

Comment:
This is still required otherwise you won't be able to request money with decimal comma (if lang is Spanish).

Reply:
@s77rt yeah, you're right! Fixed!